src/eva/diagnostics/horizontal_line.py

```diff
@@ -0,0 +1,28 @@
+from eva.eva_path import return_eva_path
+from eva.utilities.utils import get_schema, update_object
+import emcpy.plots.plots
+import os
+
+
+class HorizontalLine():
+
+    def __init__(self, config, logger, dataobj):
+
+        # Get the y value to plot
+        # -----------------------
+        yval = config['y']
+
+        # Create declarative plotting HorizontalLine object
+        # -------------------------------------------
+        self.plotobj = emcpy.plots.plots.HorizontalLine(yval)
+
+        # Get defaults from schema
+        # ------------------------
+        layer_schema = config.get('schema', os.path.join(return_eva_path(), 'defaults',
+                                  'horizontal_line.yaml'))
+        config = get_schema(layer_schema, config, logger)
+#        delvars = ['y', 'type', 'schema']
```

Comment:
remove comment

Reply:
Comment removed.   Is adding lines to an existing plot sufficient or would you prefer a stand-alone example in the test yamls (yamlae? yamlii? what is the correct plural for yaml?)?

Reply:
no need for a new test, just throw something in one of the YAMLs. Like add a line at 0 on a histogram or something.

Reply:
Done.  The result of `testIodaObsSpaceAmsuaN19.yaml`'s density plots is so:

![image](https://github.com/JCSDA-internal/eva/assets/62339196/ff09411d-094d-486d-ae70-4afe22227376)

Reply:
Looks good, thanks!!